css/_connection-info.scss

```diff
@@ -45,7 +43,6 @@
     &__upload
     {
         @extend .connection-info__icon;
-        color: $uploadConnectionIconColor;
```

Comment:
Why remove the colors?

Reply:
I didn't see the value of the colors and think they may be confusing. Because green means good connection and yellow/orange means bad connection in the color code we've chosen for the icon, so I think statically setting colors for the arrows doesn't make a lot of sense.